Library/Homebrew/test/download_strategies_spec.rb

```diff
@@ -218,6 +218,24 @@ def setup_git_repo
   end
 end
 
+describe GCSDownloadStrategy do
```

Comment:
Needs more substantial tests.

Reply:
Can you point me to some examples so I can get a better sense of what to include... 

I'm a noob with rspec and ruby 😂

Reply:
Take a look at some of the other download strategies, e.g. `GitHubGitDownloadStrategy` and see what state you can inspect.